crates/forge-runner/src/gas/report.rs

```diff
@@ -1,10 +1,14 @@
 use crate::gas::stats::GasStats;
 use cheatnet::trace_data::{CallTrace, CallTraceNode};
+use comfy_table::modifiers::UTF8_ROUND_CORNERS;
+use comfy_table::{Cell, Color, Table};
```

Comment:
Glad someone already made a lib for that and we don't have to deal with printing tables